app/assets/stylesheets/custom.css.scss

```diff
@@ -221,10 +250,18 @@ img {
   h1 {
     color: $share-yellow;
     font-size: 6em;
-    padding: 2em 2em 0;
+    padding: 1.5em 2em 0;
     text-align: center;
     text-shadow: 4px 2px $share-dark-blue;
   }
+
+  h2 {
+    color: $ghost;
```

Comment:
Properties should be ordered background-color, border-radius, color, margin, text-align